source/documentation/using_ci/travis.md

```diff
@@ -1,15 +1,19 @@
 ## Use Travis
 
-*As at April 2017, Cloudfoundry support is currently included in the latest 'edge' version of Travis. We expect it will become part of the core product soon. Likewise, our support for this is also new, so please contact our support team with any problems so we can continue to improve our documentation.*
+*As at April 2017, Cloudfoundry support is currently included in the latest 'edge' version of Travis, which means it will work with GOV.UK PaaS. We expect it will become part of the core product soon. Likewise, our support for this is also new, so please contact our support team with any problems so we can continue to improve our documentation.*
 
 In order to do this, you will need to register for a Travis account. If you link this to your GitHub account, it will be able to see all your personal code repositories as well as those of the GitHub organisations you belong to.
 
-Use the switch icons allow you to turn on the repos Travis will manage - remember that you'll need administrator access on any shared repos. This is to avoid you deploying code you shouldn't!
+Use the switch icons allow you to turn on the repos Travis will manage - remember that you'll need administrator access on any shared repos. This helps stop you deploying any code you shouldn't!
 
-You will need to know the the particular Cloudfoundry `organisation` and `space` you want Travis to deploy your app into. If you don't know, you can get these by logging into PaaS with a terminal window and using the `cf orgs` and `cf spaces` commands.
+You will need to know the the particular PaaS `organisation` and `space` you want Travis to deploy your app into. If you don't know, you can get these by logging into PaaS with a terminal window and using the `cf orgs` and `cf spaces` commands.
 
 If you haven't already done so, we also strongly recommend that you add a `name` entry to your `manifest.yml` file so that you'll know the address of your app once published. [Find out about names and URLs on PaaS](#names_routes_domains).
 
+### Plan for dedicated CI accounts
+
+Note that using your own PaaS login details within a CI service should only be a temporary measure. Dedicated user accounts should be set up for testing as soon as reasonably possible - by the end of Alpha at the latest. If you are already an `Org Manager` role on another project in Beta or Live, you should do this anyway. Find out more about [credentials for automated accounts](#credentials-for-automated-accounts).
```

Comment:
Could you describe why this advice is different (and slightly more complex) than the Jenkins advice?